federation/manifests/federation-etcd-pvc.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: {{.FEDERATION_APISERVER_DEPLOYMENT_NAME}}-etcd-claim
-  annotations:
-    volume.alpha.kubernetes.io/storage-class: "yes"
```

Comment:
>Also make sure to run federation tests.

@msau42 is this manifest used in `pull-kubernetes-federation-e2e-gce`? Not sure how to run federation tests locally.

Reply:
I would guess so.  @nikhiljindal can you confirm?